commodore/dependency_mgmt.py

```diff
@@ -22,30 +22,30 @@ def create_component_symlinks(cfg, component: Component):
     we want to access some of the file contents through the inventory.
     """
     relsymlink(
-        component.target_directory / "class",
-        f"{component.name}.yml",
-        "inventory/classes/components",
+        component.target_directory / "class" / f"{component.name}.yml",
+        P("inventory/classes/components"),
     )
+
     component_defaults_class = component.target_directory / "class" / "defaults.yml"
     if component_defaults_class.is_file():
         relsymlink(
-            component.target_directory / "class",
-            "defaults.yml",
+            component_defaults_class,
             P("inventory/classes/defaults"),
-            destname=f"{component.name}.yml",
+            f"{component.name}.yml",
```

Comment:
I prefer passing optional arguments using the keyword style, as that tends to avoid unwelcome surprises if the order of the optional arguments changes in the future.

Reply:
That was the argument I was missing.